app/components/project-card-skills.js

```diff
@@ -1,21 +1,73 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  set,
+} = Ember;
+
+/**
+  `project-card-skills` composes the given project's list of skills on the
+  project's card.
+
+  ## default usage
+
+  ```Handlebars
+  {{project-card-skills skills=project.skills}}
+  ```
+
+  @class project-card-skills
+  @module Component
+  @extends Ember.Component
+ */
+export default Component.extend({
   classNames: ['project-card-skills'],
+
+  /**
+    Returns whether or not the overflow skills are hidden.
```

Comment:
The overflow criteria actually comes from the skill itself. It's very "data up" in some sense. I would love a different way of doing it, because it's way too funky right now. Would consider reaching out to someone on the Ember slack, someone in our slack with deeper experience, or posting to StackOverflow. As of now, I'm kind of stumped for any better way to do it so I did something hacky and moved on.